tests/Doctrine/EventListener/PurgeHttpCacheListenerTest.php

```diff
@@ -61,8 +61,9 @@ public function testOnFlush(): void
         $toDeleteNoPurge = new DummyNoGetOperation();
         $toDeleteNoPurge->setId(5);
 
-        $purgerProphecy = $this->prophesize(PurgerInterface::class);
-        $purgerProphecy->purge(['/dummies', '/dummies/1', '/dummies/2', '/dummies/3', '/dummies/4'])->shouldBeCalled();
```

Comment:
you should avoid modifying existing tests as it is often a risk to break current implementations.

Reply:
How am I supposed to test the current implementation? Wouldn’t it trigger deprecations which would make tests fail?